.rubocop_todo.yml

```diff
@@ -38,53 +49,63 @@ Lint/UnusedBlockArgument:
 Lint/UnusedMethodArgument:
   Enabled: false
 
-# Offense count: 508
+# Offense count: 3
+# Configuration parameters: ContextCreatingMethods.
+Lint/UselessAccessModifier:
+  Enabled: false
```

Comment:
This cop is already enabled on develop. The checking for useless modifiers was simply improved in 0.42. 

I'm fairly certain the first two are useless:

https://git.io/viBsq
https://git.io/viBsZ

The third seems like it's just pointing to the wrong line. It's supposed to point to this private which is useless:

https://git.io/viBsE

I've opened a rubocop issue for this: https://github.com/bbatsov/rubocop/issues/3483